week2/03_log_and_register_model.py

```
@@ -0,0 +1,126 @@
+    ('regressor', LGBMRegressor(**parameters))
+])
+
+print('pipeline_created')
+
+# COMMAND ----------
+mlflow.set_experiment(experiment_name='/Shared/life-expectancy')
+
+def get_git_sha():
+    try:
+        return subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('ascii').strip()[:7]
+    except:
```

Ruff (0.7.0)

Do not use bare `except`

(E722)